proxy.yml

```diff
@@ -0,0 +1,16 @@
+config:
+  target: 'http://localhost:3009'
+  # http:
+  #   pool: 20
+  phases:
+    - duration: 180
+      arrivalRate: 10
+  defaults:
```

Comment:
I'm pretty sure you can get rid of this defaults/headers section here with no effect. I don't think you need any headers on your requests.